book/src/getting-started/5-cmake-integration.md

```diff
@@ -130,27 +132,34 @@ For this example, we are [supporting both Qt5 and Qt6 with CMake](https://doc.qt
 {{#include ../../../examples/qml_minimal/CMakeLists.txt:book_cmake_setup}}
 ```
 
-Locate [Corrosion](https://github.com/corrosion-rs/corrosion), a tool for integrating Rust libraries into CMake.
-If Corrosion is not installed, automatically download it:
+Download CXX-Qts CMake code with FetchContent:
 
 ```cmake,ignore
 {{#include ../../../examples/qml_minimal/CMakeLists.txt:book_cmake_find_corrosion}}
```

Comment:
`cmake_find_corrosion` should probably be changed ? `cmake_find_cxx_qt` or something?

Reply:
Done